cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -99,7 +99,7 @@ def counts_to_results(
     return result
 
 
-class Service(user_config.UserConfig):
+class Service(service.Service):
```

Comment:
This can be a bit confusing as to which `Service` is being imported. Let's do 
```suggestion
class Service(gss.service.Service):
```